package.json

```diff
@@ -19,10 +19,13 @@
 		"publish": "turbo build & changeset publish",
 		"changeset": "npx changeset"
 	},
+	"engines": {
+		"node": ">=18"
```

Comment:
Should this be tighter? You want people to run exactly Node.js 18, no? Or are you happy with people running 20?

Reply:
mh.... I don't know.... do we? do you see any problem if people were to use 20? (considering that we use 18 in the CI checks)

Reply:
20 (or future versions) could have breaking changes, so I think restricting to `^18` is probably correct, but up to you.

Reply:
I don't have a huge preference, I wouldn't want to bother people to force them to install `18` if they for example have `20`, but let's go for `18` and we can add `20` or whatever else later on if we want 🙂